app/models/repair.rb

```diff
@@ -8,6 +8,8 @@ def request_reference
   end
 
   def work_order_reference
-    @repair_data['orderReference']
+    return nil unless @repair_data.key?('workOrders')
+
+    @repair_data.fetch('workOrders')&.first&.fetch('workOrderReference')
```

Comment:
Could also do 

```ruby
@repair_data.fetch('workOrders', [{}]).first.fetch('workOrderReference')
```

...but it's maybe actually less readable.

Reply:
On a different note: we said in the [api standards](https://github.com/LBHackney-IT/api-standards#response-2) that:

> If the repair was created without work orders, an empty array of workOrders should be returned (rather than no key or a null value)

...so maybe it _should_ blow up if the `'workOrders'` isn't an Array? Really not sure either way.

Reply:
Yeah, I tried putting it into one line, but it felt less readable.

Whilst the API standards says this, there's no practical difference between an empty `workOrders` array and the lack of such an array.

I'd prefer to err on the side of not blowing the site up and annoying the users in this case.